src/RazorSdk/Targets/Microsoft.NET.Sdk.Razor.Compilation.targets

```diff
@@ -144,6 +144,7 @@ Copyright (c) .NET Foundation. All rights reserved.
          Features="$(Features)"
          FileAlignment="$(FileAlignment)"
          GenerateFullPaths="$(GenerateFullPaths)"
+         GeneratedFilesOutputPath="$(GeneratedFilesOutputPath)"
```

Comment:
Can you update the commit on line 15 this file was updated from?

Reply:
Ah yes -- sure!